diff -urN radmind-1.14.1/Makefile.in radmind-1.14.1-patched/Makefile.in
--- radmind-1.14.1/Makefile.in	2010-12-13 13:43:49.000000000 +1000
+++ radmind-1.14.1-patched/Makefile.in	2014-03-13 00:13:45.058136246 +1100
@@ -35,12 +35,12 @@
 
 RADMINDSYSLOG=LOG_LOCAL7
 
-INCPATH=        @CPPFLAGS@ -I${srcdir}/libsnet -I.
+INCPATH=        @CPPFLAGS@ -I.
 OPTOPTS=	@OPTOPTS@
 CC=		@CC@
 DEFS=		
-LIBS=		-lsnet @LIBS@
-LDFLAGS=	-Llibsnet/.libs @LDFLAGS@ ${LIBS}
+LIBS=		-lsnet -lsasl2 @LIBS@
+LDFLAGS=	@LDFLAGS@ ${LIBS}
 INSTALL=	@INSTALL@
 
 CFLAGS=		${DEFS} ${OPTOPTS} @CFLAGS@ ${INCPATH}
@@ -201,7 +201,7 @@
 		-D_RADMIND_TLS_CERT=\"${TLS_CERT}\" \
 		-c ${srcdir}/tls.c
 
-radmind : libsnet/libsnet.la ${RADMIND_OBJ} Makefile
+radmind : ${RADMIND_OBJ} Makefile
 	${CC} ${CFLAGS} -o radmind ${RADMIND_OBJ} ${LDFLAGS}
 
 fsdiff : ${FSDIFF_OBJ}
@@ -239,9 +239,6 @@
 
 FRC :
 
-libsnet/libsnet.la : FRC
-	cd libsnet; ${MAKE}
-
 VERSION=1.14.1
 DISTDIR=radmind-${VERSION}
 
@@ -372,7 +369,6 @@
 	sudo rm -rf ${PKGDIR}
 
 clean :
-	(cd libsnet; ${MAKE} clean)
 	rm -f *.o a.out core
 	rm -f ${TARGETS}
 	rm -rf tmp
@@ -381,4 +377,3 @@
 	rm -f config.log config.status Makefile config.h
 	rm -rf autom4te.cache
 	rm -rf .#*
-	cd libsnet; make distclean
